Index: src/App.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/App.js	(revision bbfa9c932682a039f83cec270255c6d6d64a2add)
+++ src/App.js	(revision e4a926391133fc12cade542e84b60c26ed619152)
@@ -95,7 +95,11 @@
          console.log("new grid ", this.newGridorder)
 
         // this is the only thing i need from new
-         this.newsku = this.newOrder.map(i => this.state.data[i]['sku'])
+        //  this.newsku = this.newOrder.map(i => this.state.data[i]['sku'])
+      this.newOrder = this.newOrder.map((i, seqno) => ({
+        ...this.state.data[i],
+        gridorder: this.state.data[seqno].gridorder
+      }));
          console.log("new skus ", this.newsku)
 
 
